From 8feffeb9b1f54b8d7babd531f48338c8fe4fcf5b Mon Sep 17 00:00:00 2001
From: Igor Ulyanov <disinbox@gmail.com>
Date: Thu, 6 Feb 2014 14:45:14 +0400
Subject: [PATCH 1/2] debcherry fixup patch

b43bae7 printf commented
	 - no changes against upstream or conflicts
6ea19d2 remove getchar in gglobal
	 - extra changes or conflicts
---
 src/lib/iglobal.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/lib/iglobal.c b/src/lib/iglobal.c
index f8c14b4..2092470 100644
--- a/src/lib/iglobal.c
+++ b/src/lib/iglobal.c
@@ -417,8 +417,8 @@ ttglobal gglobal()
 	ttglobal iglobal = gglobal0();
 	if(iglobal == NULL)
 	{
-		printf("ouch - no state for this thread - hit a key to exit\n");
-		getchar();
+                //printf("ouch - no state for this thread - hit a key to exit\n");
+		//getchar();
 		//let it bomb exit(-1);
 	}
 	return iglobal;
-- 
1.8.5.1

